style.css

```diff
@@ -1 +1,299 @@
 /*  Add your styling here */
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Playfair Display", serif;
+}
+/* Navigation */
+
+.navbar {
+  display: none;
+  list-style: none;
+  background-color: rgb(247, 247, 247);
+  margin: 12px 115px 5px 45px;
+  padding: 8px 7px 8px 900px;
```

Comment:
Watch out for using padding like this! this was making the screen wider than the viewport, which means there was a sidescroll bar, something we generally want to avoid! how might we move the items to the other side of the screen without using padding? If you fix this, the screen is nicely responsive!